tests/main/mounts-persist-refresh-content-snap/task.yaml

```
@@ -35,26 +36,30 @@ prepare: |
   tests.cleanup defer tests.session -u test restore
 
 execute: |
+  # Construct the mount namespace once so that we are not racing construction
+  # from background job of snap run --shell with update with snap install of
```

use of `snap run --shell` was removed
```suggestion
  # from background job with update with snap install of
```

---

Oh, I should have noticed that (waves hand at the pair of crushed glasses)